rest-api/recipe/models/recipe-models.js

```diff
@@ -0,0 +1,107 @@
+const db = require('../../../database/dbConfig')
+module.exports = {
+    getRecipes,
+    getRecipeById
+}
+
+async function getRecipes() {
+    const recipes = await db('recipes as r')
+        .join(
+            'recipe_images as rim',
+            'rim.recipe_id',
+            'r.id'
```

Comment:
I would replace `r. ` with `recipes.` for readability since you're not joining tables with similar names